pwndbg/commands/hexdump.py

```diff
@@ -23,17 +24,32 @@
                          64,
                          'number of bytes printed by hexdump command')
 
-parser = argparse.ArgumentParser(description='Hexdumps data at the specified address (or at $sp)')
-parser.add_argument('address', nargs='?', default='$sp',
-                    help='Address to dump')
+def address_or_module_name(s):
```

Comment:
Hmm this is solved in vmmap as https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/vmmap.py#L24-L38

It would probably be better to keep just one such function in the whole codebase.

Can we move it e.g. to pwndbg/commands/__init__.py and reuse it in here and in vmmap?

Reply:
Good point. I'l try to do that. However it looks like I can't use it directly since it returns pages in both cases and here we need page only in one case.

Reply:
Hmm good point; though, what if a given objfile has many pages? we just go with the first one?

Reply:
Yes, that was my intention. I.e. to usages like `hexdump stack` or `hexdump heap` will start from the beginning.

Reply:
I think it is reasonable, let's merge it :).